sylt-std/src/sylt.rs

```diff
@@ -456,7 +461,7 @@ sylt_macro::extern_function!(
     "sylt_std::sylt",
     dot,
     ? "Computes the scalar product",
-    -> "fn (float, float) -> float",
+    -> "fn (float, float), (float, float) -> float",
```

Comment:
This one could be a num as well. But no need to change it now.

Reply:
We'd need to change the function, the break up vec function returns floats.